basic-auth/routes/signup.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express');
+const authRoutes = express.Router();
+
+authRoutes.get('/signup', (req, res, next) => {
+  res.render('auth/signup');
+});
+
+// User model
+const User           = require("../models/user");
+
+// BCrypt to encrypt passwords
+const bcrypt         = require("bcrypt");
+const bcryptSalt     = 10;
```

Comment:
The organization of the file here could be a bit better. These two lines should move up